src/lightning/pytorch/strategies/xla.py

```diff
@@ -92,14 +92,8 @@ def root_device(self) -> torch.device:
 
         return xm.xla_device()
 
-    @property
-    def local_rank(self) -> int:
-        return self.cluster_environment.local_rank() if self.cluster_environment is not None else 0
-
     def connect(self, model: "pl.LightningModule") -> None:
-        import torch_xla.distributed.xla_multiprocessing as xmp
-
-        self.wrapped_model = xmp.MpModelWrapper(_LightningModuleWrapperBase(model))
+        self.wrapped_model = _LightningModuleWrapperBase(model)
```

Comment:
`connect` is called in spawned processes, so there is no point to wrap it with `xmp.MpModelWrapper`

Reply:
Let's leave a comment

```suggestion
        # this is called in the spawned process, so no need to use `xmp.MpModelWrapper`
        self.wrapped_model = _LightningModuleWrapperBase(model)
```